data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/range_action/PstWithRange.java

```diff
@@ -48,17 +43,16 @@
      *
      * @param networkElement: PST element to modify
      */
-    @JsonCreator
-    public PstWithRange(@JsonProperty("id") String id,
-                        @JsonProperty("name") String name,
-                        @JsonProperty("operator") String operator,
-                        @JsonProperty("usageRules") List<UsageRule> usageRules,
-                        @JsonProperty("ranges") List<Range> ranges,
-                        @JsonProperty("networkElement") NetworkElement networkElement) {
-        super(id, name, operator, usageRules, ranges, networkElement);
+    public PstWithRange(String id, String name, String operator, List<UsageRule> usageRules, List<Range> ranges,
+                        NetworkElement networkElement, String groupId) {
+        super(id, name, operator, usageRules, ranges, networkElement, groupId);
         initAttributes();
     }
 
+    public PstWithRange(String id, String name, String operator, List<UsageRule> usageRules, List<Range> ranges, NetworkElement networkElement) {
+        super(id, name, operator, usageRules, ranges, networkElement);
+    }
```

Comment:
I think an `initAttributes()` is needed here (all other constructors have one)

Reply:
done